srcpkgs/spek-alternative/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'spek-alternative'
+pkgname=spek-alternative
+version=0.8.2.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake gettext-devel intltool libtool pkg-config wxWidgets-common"
```

Comment:
What does it need from `wxWidgets-common` in `hostmakedepends`?

Reply:
Yep